app/javascript/js/controllers/item_select_all_controller.js

```diff
@@ -43,6 +43,10 @@ export default class extends Controller {
       this.selectAllOverlay(allSelected)
       this.resetUnselected()
     }
+
+    const resourceIds = JSON.parse(this.element.dataset.selectedResources).join(',')
+    const allSelectedValue = ''
+    this.updateLinks(resourceIds, allSelectedValue)
```

Comment:
Let's avoid unnecessary const initialization by giving `allSelected` a default value on the function definition

```suggestion
    this.updateLinks(resourceIds)
```